remote-trait-object/src/service/serde_support.rs

```diff
@@ -80,8 +80,9 @@ impl<T: ?Sized + Service> SRwLock<T> {
     }
```

Comment:
In the commit mesesage: Edit old comments on **prot**_thread_local -> Edit old comments on **port**_thread_local

Reply:
done